tools/its.py

```diff
@@ -204,7 +207,7 @@ def get_chromedriver(self, headless):
 
         chrome_options.headless = headless
 
-        return Chrome(self.get_chromedriver_path(), options=chrome_options)
+        return Chrome(options=chrome_options)
```

Comment:
warum wird der Pfad hier rausgenommen?

Reply:
Zeile 16 im Docker File war nicht erfolgreich. Im docker Container ist der chrome driver schon da und geht ohne. Aber ja das muss man besser machen.

Reply:
Hast du die Umgebungsvariable `VACCIPY_CHROMEDRIVER` verwendet?

Reply:
> Hast du die Umgebungsvariable `VACCIPY_CHROMEDRIVER` verwendet?

Ist ja im Dockerfile prinzipiell drin.

Reply:
Jetzt bin ich wieder, wenn die Zeile wieder die Methode aufruft.

`2021-06-10 13:52:46.857   impfterminservice   [err] [*C4Q2 | 68163, 69124, 69469]   (unknown error: DevToolsActivePort file doesn't exist)
(The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)`